src/style.css

```diff
@@ -1,3 +1,120 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  padding: 10px 50px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  text-align: left;
+}
+
+@media (min-width: 1024px) and (max-width: 1199px) {
+  .nav_list {
+    list-style: none;
+    padding: 0;
+    margin-left: 283px;
+    display: flex;
+    align-items: center;
+  }
+
+  .nav_item {
+    margin: 0 20;
```

Comment:
The margin value `0 20` is missing a unit for the second value. It should be `0 20px` to ensure consistent rendering across different browsers.